packages/create-rise/src/cli.ts

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+import fs from 'node:fs'
+import path from 'node:path'
+
+import inquirer from 'inquirer'
+
+const cwd = process.cwd()
+
+const defaultProjectName = 'rise-project'
+
+async function prompt() {
+  let targetDir = ''
```

Comment:
I know this is totally out of scope for this PR, but maybe we print beautiful Rise logo here (and in CLI start command), like React Native! 😆

CC: @ericvicenti